src/components/org-users/permissions.njk

```diff
@@ -163,46 +175,67 @@
+                  {% if (values.space_roles[space.metadata.guid].managers.desired == '1') %}
+                    checked
+                  {% endif %}
+                  value="1">
+                <label class="govuk-label govuk-checkboxes__label" for="space_roles[{{ space.metadata.guid }}][managers]">
+                  <span>Space manager</span>
+                </label>
+              </div>
+            </div>
+          </div>
         </td>
         <td class="govuk-table__cell ">
-          <input type="hidden" value="{{ values.space_roles[space.metadata.guid].developers }}" name="space_roles[{{ space.metadata.guid }}][developers][current]">
-          {{ govukCheckboxes({
-            id: "space_roles[" + space.metadata.guid + "][developers]",
-            name: "space_roles[" + space.metadata.guid + "][developers][desired]",
-            items: [
-              {
-                value: "1",
-                html: "<span>Space developer</span>",
-                checked: values.space_roles[space.metadata.guid].developers
-              }
-            ]
-          }) }}
+          <input type="hidden" value="{{ values.space_roles[space.metadata.guid].developers.current }}" name="space_roles[{{ space.metadata.guid }}][developers][current]">
+          <div class="govuk-form-group">
+            <div class="govuk-checkboxes">
+              <div class="govuk-checkboxes__item">
+                <input
+                  class="govuk-checkboxes__input"
+                  id="space_roles[{{ space.metadata.guid }}][developers]"
+                  name="space_roles[{{ space.metadata.guid }}][developers][desired]"
+                  type="checkbox"
+                  {% if (values.space_roles[space.metadata.guid].developers.desired == '1') %}
+                    checked
+                  {% endif %}
+                  value="1">
+                <label class="govuk-label govuk-checkboxes__label" for="space_roles[{{ space.metadata.guid }}][developers]">
+                  <span>Space developer</span>
+                </label>
+              </div>
+            </div>
+          </div>
         </td>
         <td class="govuk-table__cell ">
-          <input type="hidden" value="{{ values.space_roles[space.metadata.guid].auditors }}" name="space_roles[{{ space.metadata.guid }}][auditors][current]">
-          {{ govukCheckboxes({
-            id: "space_roles[" + space.metadata.guid + "][auditors]",
-            name: "space_roles[" + space.metadata.guid + "][auditors][desired]",
-            items: [
-              {
-                value: "1",
-                html: "<span>Space auditor</span>",
-                checked: values.space_roles[space.metadata.guid].auditors
-              }
-            ]
-          }) }}
+          <input type="hidden" value="{{ values.space_roles[space.metadata.guid].auditors.current }}" name="space_roles[{{ space.metadata.guid }}][auditors][current]">
+          <div class="govuk-form-group">
```

Comment:
This bit is repeated almost identically seven times - maybe worth pulling it out into a macro and just passing in the bits that change, so we'd call something like:

```
{{ roleCheckbox("org_roles", "managers", "Org manager", isCheckboxDisabled) }}
...
{{ roleCheckbox("org_roles", "billing_managers", "Billing manager", isCheckboxDisabled) }}
...
{{ roleCheckbox("org_roles", "developers", "Org developer", false) }}
...
{{ roleCheckbox("org_roles", "auditors", "Org auditor", false) }}
...
{{ roleCheckbox("space_roles", "managers", "Space manager", false) }}
...
{{ roleCheckbox("space_roles", "developers", "Space developer", false) }}
...
{{ roleCheckbox("space_roles", "auditors", "Space auditor", false) }}
```